SnapWar/ViewController.swift

```diff
@@ -8,11 +8,12 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UICollectionViewController {
```

Comment:
let's leave this as UIViewController, unless you wanted to make this a collection view?

Reply:
also, let's just add the other view controllers as we build them, no need to pre-populate (just in case we don't end up needing them)